src/components/PressableWithSecondaryInteraction/index.tsx

```diff
@@ -1,76 +1,73 @@
 import React, {forwardRef, useEffect, useRef} from 'react';
-import _ from 'underscore';
+import {GestureResponderEvent} from 'react-native';
+import {PressableRef} from '@components/Pressable/GenericPressable/types';
 import PressableWithFeedback from '@components/Pressable/PressableWithFeedback';
 import * as DeviceCapabilities from '@libs/DeviceCapabilities';
 import styles from '@styles/styles';
 import * as StyleUtils from '@styles/StyleUtils';
-import * as pressableWithSecondaryInteractionPropTypes from './pressableWithSecondaryInteractionPropTypes';
-
-/**
- * This is a special Pressable that calls onSecondaryInteraction when LongPressed, or right-clicked.
- */
-
-function PressableWithSecondaryInteraction({
-    children,
-    inline,
-    style,
-    enableLongPressWithHover,
-    withoutFocusOnSecondaryInteraction,
-    preventDefaultContextMenu,
-    onSecondaryInteraction,
-    onPressIn,
-    onPress,
-    onPressOut,
-    activeOpacity,
-    forwardedRef,
-    ...rest
-}) {
-    const pressableRef = useRef(null);
-
-    /**
-     * @param {Event} e - the secondary interaction event
-     */
-    const executeSecondaryInteraction = (e) => {
+import PressableWithSecondaryInteractionProps from './types';
+
+/** This is a special Pressable that calls onSecondaryInteraction when LongPressed, or right-clicked. */
+function PressableWithSecondaryInteraction(
+    {
+        children,
+        onPress,
+        onPressOut,
```

Comment:
Why do you still have `onPressOut` here if it's not being used, also I'm curious why eslint doesn't give an error

Reply:
Removed!